posthog/api/github.py

```diff
@@ -0,0 +1,162 @@
+import base64
+import requests
+from typing import Any
+
+from django.apps import apps
+from django.core.cache import cache
```

Comment:
Cache will be filled once per process, and we have hundreds of API pods. Maybe there's a better approach to caching this to avoid hitting Github's API this often? I know it's a static file, but maybe we could use Redis instead?

Reply:
Wow, I thought this was using Redis behind the scenes. Redis was very much my intention - will update this.